frontend/src/components/widgets/DataFrame/columns/utils.test.ts

```diff
@@ -426,3 +429,70 @@ describe("truncateDecimals", () => {
     }
   )
 })
+
+describe("formatMoment", () => {
+  beforeAll(() => {
+    jest.useFakeTimers("modern")
+    jest.setSystemTime(new Date("2023-04-28T00:00:00Z"))
+    timezoneMock.register("UTC")
+  })
+
+  afterAll(() => {
+    jest.useRealTimers()
+    timezoneMock.unregister()
+  })
+
+  it.each([
+    [
+      "yyyy-MM-dd HH:mm:ss zzz",
+      moment.utc("2023-04-27T10:20:30Z"),
+      "2023-04-27 10:20:30 UTC",
+    ],
+    [
+      "yyyy-MM-dd HH:mm:ss zzz",
+      moment.utc("2023-04-27T10:20:30Z").tz("America/Los_Angeles"),
+      "2023-04-27 03:20:30 PDT",
+    ],
+    [
+      "yyyy-MM-dd HH:mm:ss xxx",
+      moment.utc("2023-04-27T10:20:30Z").tz("America/Los_Angeles"),
+      "2023-04-27 03:20:30 -07:00",
+    ],
+    [
+      "yyyy-MM-dd HH:mm:ss xxx",
+      moment.utc("2023-04-27T10:20:30Z").utcOffset("+04:00"),
+      "2023-04-27 14:20:30 +04:00",
+    ],
+    ["yyyy-MM-dd", moment.utc("2023-04-27T10:20:30Z"), "2023-04-27"],
+    [
+      "MMM do, yyyy 'at' h:mm aa",
+      moment.utc("2023-04-27T15:45:00Z"),
+      "Apr 27th, 2023 at 3:45 PM",
+    ],
+    [
+      "MMMM do, yyyy xxxxx",
+      moment.utc("2023-04-27T10:20:30Z").utcOffset("-02:30"),
+      "April 27th, 2023 -02:30",
+    ],
+    // Distance:
+    ["distance", moment.utc("2023-04-10T20:20:30Z"), "2 weeks ago"],
+    ["distance", moment.utc("2021-04-10T20:20:30Z"), "2 years ago"],
+    ["distance", moment.utc("2023-04-27T23:59:59Z"), "1 second ago"],
+    ["distance", moment.utc("2023-04-20T00:00:00Z"), "last week"],
+    ["distance", moment.utc("2023-05-27T23:59:59Z"), "in 4 weeks"],
+    ["relative", moment.utc("2023-04-30T15:30:00Z"), "Sunday at 3:30 PM"],
+    // Relative:
+    [
```

Comment:
Nope, I think this should stay that way since i set above:

```
jest.useFakeTimers("modern")
jest.setSystemTime(new Date("2023-04-28T00:00:00Z"))
timezoneMock.register("UTC")
```

Only thing I wasn't really able to set was the `locale`, but I don't think that running jest tests will use any other than the default locale.

Reply:
Ah gotcha. I slightly missed that. thanks.

Reply:
I  changed the date to 2022, so it's a bit more obvious that it's working fine for the test